### PR TITLE
bookworm: unstabled-2018-11-19 -> 1.1.1

### PR DESCRIPTION
**Pull request by flokli**
This bumps bookworm to the latest release, now using meson as build
system.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/office/bookworm/default.nix [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+    rev = version;
+    sha256 = "0qzwvlwv3nbgzqxxd7kd4rlvld7k03ivmyh099rzm6z00mm4y031";
   };
 
   nativeBuildInputs = [
     bash
     gobject-introspection
     libxml2
-    meson
+    cmake
```
Comment (worldofpeace) [inline pkgs/applications/office/bookworm/default.nix]: Now using meson?

**Comment by flokli** [comment]
Now using cmake, sorry. There's no more meson.build file around either.

**Comment by worldofpeace** [comment]
This on an unstable revision because we needed https://github.com/babluboy/bookworm/commit/07fbf07ccf9591f69267b5783581b26ca27ff313 and I wanted the meson build

See:
* https://github.com/NixOS/nixpkgs/pull/50742
* #49560
* https://github.com/babluboy/bookworm/pull/220

Why downgrade?

**Comment by flokli** [comment]
Argh, I got confused with versioning, assumed this was an upgrade. Sorry for the noise.

**Comment by worldofpeace** [comment]
TBH the versions confused me too :smile: 


Screenshot from 2019-04-08 03 17 10

The conflicting information, not to mention the dates.

**Pull request closed without merging**